interceptors/retry/examples_test.go

```diff
@@ -88,3 +91,39 @@ func ExampleWithPerRetryTimeout() {
 		retry.WithMax(3),
 		retry.WithPerRetryTimeout(1*time.Second))
 }
+
+// scale duration by a factor.
+func scaleDuration(d time.Duration, factor float64) time.Duration {
+	return time.Duration(float64(d) * factor)
+}
+
+func TestJitterUp(t *testing.T) {
```

Comment:
Let's move this out of examples to separate test file. I think it will work, just might be good to separate Examples from tests 🤗 Not a biggy.

Reply:
Done, moved to `retry_test.go`. Let me know if this works!